src/functions/identical-commit-check.js

```diff
@@ -41,15 +41,15 @@ export async function identicalCommitCheck(octokit, context, environment) {
   var deploymentId
   for (const deployment of deploymentsData) {
     if (deployment.payload.type === 'branch-deploy') {
-      latestDeploymentSha = deployment.sha
```

Comment:
This variable wasn't defined so I figured it was probably intended to be `latestDeploymentTreeSha` instead.

Reply:
:+1: Yep, good catch!